pkg/kube/controllers/extendedstatefulset/reconciler.go

```diff
@@ -211,6 +232,193 @@ func (r *ReconcileExtendedStatefulSet) Reconcile(request reconcile.Request) (rec
 	return reconcile.Result{}, nil
 }
 
+// alterOperatorStatefulSet creates the operator statefulset for persistent volume claim creation
+func (r *ReconcileExtendedStatefulSet) alterOperatorStatefulSet(ctx context.Context, actualVersion int, desiredVersion int, exStatefulSet *essv1a1.ExtendedStatefulSet, actualStatefulSet *v1beta2.StatefulSet) error {
```

Comment:
You're adding a lot of code to this file - is there a way you can bundle these up in a different file (e.g. `volumes.go`)?

Reply:
Yes. We can. Done.